vars/groups.groovy

```diff
@@ -4,40 +4,24 @@ import groovy.json.JsonOutput
 def createGroup(String groupName, List groupMembers, String groupType) {
    return JsonOutput.toJson([
       name: groupName,
-      members: groupMembers,
+      discordID: groupDiscordID,
+      swarmID: groupSwarmID,
       type: groupType
    ]);
 }
 
-def getGroupMembers(groups, String groupName) {
-    for(group in groups) {
-        if(group.name == groupName) {
-            return group.members;
-        }
-    }
-
-    log.error("Tried to get group members for group '${groupName}', but this group couldn't be found!");
-    return [];
-}
-
-def getGroupType(groups, String groupName) {
-    for(group in groups) {
-        if(group.name == groupName) {
-            return group.type;
-        }
-    }
-
-    log.error("Tried to get group type for group '${groupName}', but this group couldn't be found!");
-    return [];
-}
-
 def mentionGroup(groups, String groupName) {
-    def groupMembers = getGroupMembers(groups, groupName);
-    def groupType = getGroupType(groups, groupName);
-
-    if(!groupMembers && !groupType) {
-        return '';
-    }
+   def groupType = ""
+   def discordID = ""
+   def groupsParsed = new JsonSlurper().parseText(groups)
```

Comment:
Doesn't `loadGroups` already parse the groups?

If you're passing the full group json into this function, maybe split `loadGroups` up into `loadGroupsFromFile` and `loadGroupsFromString`, or have `loadGroups` return the json string rather than a parsed version. (This would also mean that `saveGroups`  shouldn't convert the groups to json, as they would already be in that format).

Reply:
Oh yeah missed that function. I suppose we can remove loadGroups (and pass file as string as argument) or convert loadGroups to just reading in the file and parsing it using JsonSlurper and return the JsonObject.